src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
+import { UserInfoType } from '../../type/user';
+import './UserInfo.scss';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+export const UserInfo: React.FC<UserInfoType> = ({
+  user: {
+    email,
+    name,
+  },
+}) => (
+  <p>
+    {' Posted by  '}
```

Comment:
Remove unnecessary spaces in the 'Posted by' text. One in the beginning and one in the end (there is 2)